pkg/archive/tar.go

```diff
@@ -46,11 +46,25 @@ type TarArchiveStabilizer struct {
 	Func func(*TarArchive)
 }
 
+func (t TarArchiveStabilizer) Stabilize(arg any) {
+	if t.Func == nil {
```

Comment:
ah I just meant let it panic when you try to call nil on line 53. the extra code likely doesn't cut appreciable time off of the debug process.

Reply:
Cool. Made the change. `panic` statement just made it more explicit where the error is but the stack trace is also  clear.